internal/store/builder.go

```diff
@@ -34,6 +34,7 @@ import (
 	extensions "k8s.io/api/extensions/v1beta1"
 	policy "k8s.io/api/policy/v1beta1"
 	storagev1 "k8s.io/api/storage/v1"
+	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Comment:
I think we can use `storagev1` for VolumeAttachment. It has been promoted to v1 since 1.13 if I am not mistaken.

Reply:
It should even support the watching and listing of VolumeAttachments created under `v1beta1`

Reply:
Yes, it graduated to GA in 1.13 (as also stated in the PR). I started with `storagev1` originally, but it failed in a Kubernetes 1.14 cluster -- kube-state-metrics got v1beta1 objects returned and paniced.

On the other hand, following the compatibility chart from the README we need to support Kubernetes 1.11? In this case, only `storagev1beta1` is available.